docs/guide/advanced/navigation-failures.md

```diff
@@ -0,0 +1,60 @@
+# Navigation Failures
+
+> New in 3.4.0
+
+When using `router-link`, Vue Router internally calls `router.push` to trigger a navigation. Depending on the current location and existing [Navigation Guards](./navigation-guards.md), this navigation might end up in a new page being shown, but there are a couple of situations where we will stay on the same page:
+
+- We are already on the page we are trying to go to
+- A navigation guard aborts the navigation by calling `next(false)`
+- A navigation guard throws an error or calls `next(new Error())`
+
+When using a regular `router-link`, **none of these failures will log an error**. However, if you are using `router.push` or `router.replace`, you might come across an _"Uncaught (in promise) Error"_ message followed by a more specific message in your console. Let's understand how to differentiate _Navigation Failures_.
```

Comment:
```suggestion
When using a `router-link` element, **none of these failures will log an error**. However, if you are using `router.push` or `router.replace`, you might come across an _"Uncaught (in promise) Error"_ message followed by a more specific message in your console. Let's understand how to differentiate _Navigation Failures_.
```

Reply:
should just omit it? or maybe say _component_?

Reply:
Works for me!